lib/index.ts

```diff
@@ -21,14 +21,14 @@ function getAllSwears(): Array<swearVariants> {
         [`cibole`, `cibolac`],
         [`enfant d'chienne`],
         [`verrat`],
-        [`marde`, `maudite marde`, `mangeux d'marde`],
+        [`marde`, `maudite marde`, `mangeux d'marde`, `gros tas d'marde`],
```

Comment:
I'm not sure about this one as it doesn't seem to really work as a standalone swear? Seems like it's used pretty commonly as an insult, but I don't think we really just say "Gros tas d'marde" when we're angry.